docker-compose.yml

```diff
@@ -19,6 +19,35 @@ services:
         target: /library
         consistency: delegated
 
+  ghsync:
+    image: smacker/ghsync
+    entrypoint: ['/bin/sh']
+    # first sleep - wait for db to be created
+    # second sleep - wait for rabbit to be available (takes A LOT of time)
+    # we are going to remove rabbit dependency but for postgres
+    # we need to use something like https://github.com/vishnubob/wait-for-it
+    command: ['-c', 'sleep 30s && ghsync migrate && sleep 30s && ghsync sync']
+    depends_on:
+      - metadatadb
+      - rabbitmq
+    environment:
+      GHSYNC_ORG: ${GITHUB_ORGANIZATION:-}
+      GHSYNC_TOKEN: ${GITHUB_TOKEN:-}
+      GHSYNC_POSTGRES_DB: metadata
```

Comment:
I'd suggest the same than in https://github.com/src-d/ghsync/pull/25#discussion_r292950539
I'd let the user choose it, and use `metadata` as a default value

Reply:
we don't allow to choose names of maindb or gitbase from the cli. Why should we allow to choose a name for metadata? 
And of course, the user still can change all the names in docker-compose.yml manually.

Reply:
imo using vars also helps to understand the relations between different parts, but not a requirement to approve this PR, that LGTM

Reply:
makes sense to use variables to avoid repeated hard-coded values in different containers! 👍 
shouldn't be done in the PR though. I created an issue: https://github.com/src-d/sourced-ce/issues/62